woocommerce/woocommerce
# Edit contributing docs readme

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Edits language and links in the contribution section of the monorepo README. Edits Contributing to Woo README in the docs folder to add links and context for contribution. I decided to keep the original files in their `.github ` folder to follow GitHub's standards in their own folder. Original docs do feel cyclical, so I do believe there is opportunity to consolidate those docs as well, but would prefer to do that with an eng. 

Closes # .



### How to test the changes in this Pull Request:

1. Docs manifest is updated
2. Check the files in markdown for any typos or errant links. Make sure there are no apostrophes, as the docs site still does not read them, even in markdown.
